app/models/taxonomies/location.rb

```diff
@@ -15,8 +15,9 @@ class Location < Taxonomy
 
   scope :completer_scope, ->(opts) { my_locations }
 
-  scope :my_locations, lambda {
-    conditions = User.current.admin? ? {} : sanitize_sql_for_conditions([" (taxonomies.id in (?))", User.current.location_and_child_ids])
+  scope :my_locations, lambda { |user = nil|
```

Comment:
Nitpick, move the `user ||= User.current` to the lambda args `|user =  User.current|`, if you want to provide a default I think that's the usual way